tests/Test.ADAL.NET.Unit/AdalDotNetTests.cs

```diff
@@ -1494,5 +1496,85 @@ public void Initialize()
             }
         }
 
+        [TestMethod]
+        [Description("Telemetry tests")]
+        public void Telemetry()
+        {
+            Microsoft.IdentityModel.Clients.ActiveDirectory.Telemetry telemetry =
+            Microsoft.IdentityModel.Clients.ActiveDirectory.Telemetry.GetInstance();
+            Assert.IsNotNull(telemetry);
+
+            string requestIDOne = telemetry.RegisterNewRequest();
+            Assert.IsNotNull(requestIDOne);
+
+            telemetry.StartEvent(requestIDOne, "acquire_token");
+            string requestIDTwo = telemetry.RegisterNewRequest();
+            Assert.AreEqual(telemetry.EventsStored(), 1);
+
+            DefaultEvent testDefaultEvent = new DefaultEvent("random_event");
+            Assert.IsNotNull(DefaultEvent.ApplicationName);
+            Assert.IsNotNull(DefaultEvent.ApplicationVersion);
+            Assert.AreEqual(testDefaultEvent.Tenant,"tid");
+            Assert.AreEqual(testDefaultEvent.PasswordChangeUrl, "pwd_url");
+            Assert.AreEqual(testDefaultEvent.PasswordExpiration, "pwd_exp");
+            Assert.AreEqual(testDefaultEvent.Idp,"idp");
+            Assert.AreEqual(testDefaultEvent.Issuer,"iss");
```

Comment:
just curious about how this is working. you did not do a token request, so how do you have the idtoken params